src/routes/load/components/Layout.tsx

```diff
@@ -48,7 +48,7 @@ const Layout = ({ network, onLoadSafeSubmit, provider, userAddress }: LayoutProp
           <IconButton disableRipple onClick={back} style={iconStyle}>
             <ChevronLeft />
           </IconButton>
-          <Heading tag="h2">Load existing Safe</Heading>
+          <Heading tag="h2">Add existing account</Heading>
```

Comment:
Same case here. They should be loading a Gnosis Safe, not just an account

Reply:
I think it would be fine to call it "Add existing Gnosis Safe account" but that would be too long. I generally see the point you are raising. However, the description below indicates that this is about Gnosis Safe accounts vs. any other accounts, and latest in the next step this becomes clear.